types/echarts/options/series/sankey.d.ts

```diff
@@ -183,6 +192,17 @@ declare namespace echarts {
              */
             nodeGap?: number;
 
+            /**
+             * Alignment of nodes in the graph.
+             *
+             * May be 'left', 'right' or 'justify'
+             *
+             * @default
+             * justify
+             * @see https://echarts.apache.org/en/option.html#series-sankey.nodeAlign
+             */
+            nodeAlign?: string;
```

Comment:
`nodeAlign` is also not available in the doc. But I tried with the demo and it works. Maybe you should make another PR to the doc: https://github.com/apache/incubator-echarts-doc

Reply:
i have added `name` and `nodeAlign` to the documentation  to support these links in the typings file:

https://github.com/apache/incubator-echarts-doc/pull/107

`name` under series will appear in the tooltip parameters as `seriesName`, same as everywhere else:, and also in the legend, if you add one: this small change to the sankey example demonstrates:

```
option = {
    tooltip: {
        formatter: (params) => params.seriesName
    },
    legend: {},
    series: [{
        type: 'sankey',
        layout: 'none',
        name: "flow_diagram",
        focusNodeAdjacency: 'allEdges',
      
        nodeAlign: 'left',
        data: [{
            name: 'a'
        }, {
            name: 'b'
        }, {
            name: 'a1'
        }, {
            name: 'a2'
        }, {
            name: 'b1'
        }, {
            name: 'c'
        }],
        links: [{
            source: 'a',
            target: 'a1',
            value: 5
        }, {
            source: 'a',
            target: 'a2',
            value: 3
        }, {
            source: 'b',
            target: 'b1',
            value: 8
        }, {
            source: 'a',
            target: 'b1',
            value: 3
        }, {
            source: 'b1',
            target: 'a1',
            value: 1
        }, {
            source: 'b1',
            target: 'c',
            value: 2
        }]
    }]
    
};
```